airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/PostgresJdbcSourceAcceptanceTest.java

```diff
@@ -523,4 +533,128 @@ public void testUserHasNoPermissionToDataBase() throws Exception {
     assertTrue(status.getMessage().contains("State code: 42501;"));
   }
 
+  @Override
+  protected void incrementalCursorCheck(
```

Comment:
Remind me again - this is so the `DbStreamState` can have the state_type field correct>

Reply:
Yep! This is to make sure all the actual and expected states have the state_type key.